oak_functions/wasm/src/lib.rs

```diff
@@ -563,15 +479,10 @@ where
             &format!("running Wasm module completed with result: {:?}", result),
         );
 
-        // Terminate the extensions.
-        store
-            .data_mut()
-            .extensions
-            .values_mut()
-            .try_for_each(|e| e.terminate())?;
+        let response_bytes = response.lock().clone();
```

Comment:
It would be nice as a future improvement to not have to clone the bytes and remove the spinlock.